.github/workflows/continuous-integration-workflow.yml

```
@@ -76,10 +76,9 @@ jobs:
       - name: maybe_disable_death_tests
         if: ${{ matrix.distro == 'fedora:rawhide' }}
         run: echo "GTEST_FILTER=-*DeathTest*" >> $GITHUB_ENV
-# Re-enable when latest is F37+
-#      - name: maybe_use_flang
-#        if: ${{ matrix.cxx == 'clang++' && startsWith(matrix.distro,'fedora:') }}
-#        run: echo "FC=flang" >> $GITHUB_ENV
+      - name: maybe_use_flang
+        if: ${{ matrix.cxx == 'clang++' && startsWith(matrix.distro,'fedora:') }}
+        run: echo "FC=flang" >> $GITHUB_ENV
```

```suggestion
        run: echo "FC=flang-new" >> $GITHUB_ENV
```
Don't ask me why......